src/containers/project-container/project-container.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { useEffect, useState } from 'react';
+import { AxiosResponse } from 'axios';
+import { apiFetch } from '../../shared/axios';
+import { Project, WorkPackage, apiRoutes } from 'utils';
+import ProjectDetails from '../../components/project-details/project-details';
+import WorkPackageSummary from '../../components/work-package-summary/work-package-summary';
+import styles from './project-container.module.css';
+import { wbsPipe } from '../../shared/pipes';
```

Comment:
I recommend putting this import line right after the other shared folder import, and place both of the shared folder imports after the utils package import.